app/App.js

```diff
@@ -120,13 +120,14 @@ class App extends React.Component {
   }
 
   render () {
+    const { state } = this;
     return (
       <div>
-        <TopBar {...this.state}
-          onPeriodChange={this._changePeriod.bind(this)}
-          onProfileChange={this._changeProfile.bind(this)} />
+        <TopBar {...state}
+          onPeriodChange={this._changePeriod}
+          onProfileChange={this._changeProfile} />
```

Comment:
Small thing, but if you've got multiple props which needs to break.
Then break all of them.

from
```jsx
        <TopBar {...state}
          onPeriodChange={this._changePeriod}
          onProfileChange={this._changeProfile} />
```

to 
```jsx
        <TopBar 
          {...state}
          onPeriodChange={this._changePeriod}
          onProfileChange={this._changeProfile} 
        />
```